cassandra/metadata.py

```diff
@@ -656,7 +656,7 @@ def as_cql_query(self):
         ret = "CREATE KEYSPACE %s WITH replication = %s " % (
             protect_name(self.name),
             self.replication_strategy.export_for_schema())
-        return ret + (' AND durable_writes = %s;' % ("true" if self.durable_writes else "false"))
+        return ret + (' AND durable_writes = %s' % ("true" if self.durable_writes else "false"))
```

Comment:
Now we're missing a semicolon after the keyspace cql in KeyspaceMetadata.export_as_string